modules/sbo-creating-appropriate-clusterrole-and-clusterrolebinding-resources.adoc

```diff
@@ -0,0 +1,46 @@
+[id="sbo-creating-appropriate-clusterrole-and-clusterrolebinding-resources_{context}"]
+= Creating appropriate ClusterRole and ClusterRoleBinding resources
+
+You can allow the {servicebinding-title} to `read`, `watch`, and `list` the custom resources (CRs) of Crunchy PostgreSQL Operator by creating appropriate `ClusterRole` and `ClusterRoleBinding` resources.
+
+[discrete]
+.Procedure
+
+. Initialize the Role-based access control (RBAC) support by using  the custom shell script:
++
+----
+$ oc apply -f - << EOD
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: postgrescluster-reader
+rules:
+- apiGroups: ["postgres-operator.crunchydata.com"]
+  resources: ["postgresclusters"]
+  verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: service-binding-operator-postgrescluster-reader
+subjects:
+- kind: ServiceAccount
+  name: service-binding-operator
+  namespace: $OPERATOR_NAMESPACE
+roleRef:
+  kind: ClusterRole
+  name: postgrescluster-reader
+  apiGroup: rbac.authorization.k8s.io
+EOD
+----
++
+The output verifies that the appropriate `ClusterRole` and `ClusterRoleBinding` resources are created:
++
+.Example output
+----
+clusterrole.rbac.authorization.k8s.io/postgrescluster-reader created
+clusterrolebinding.rbac.authorization.k8s.io/service-binding-operator-postgrescluster-reader created
+----
+
+You can now deploy the sample application and connect it to the database service.
```

Comment:
This whole module is obsolete and should be removed since crunchy postgres is supported by default via #1043. @pedjak please confirm